templates/_github/workflows/test-and-release.yml.ts

```diff
@@ -105,92 +113,94 @@ ${(adapterTestOS.includes("windows-latest")) ? (`
             - name: Run integration tests (windows only)
               if: startsWith(runner.OS, 'windows')
               run: set DEBUG=testing:* & npm run test:integration`) :""}
-`) : ""}
+`) : ""}${deploy ? "" : (`
 # TODO: To enable automatic npm releases, create a token on npmjs.org 
 # Enter this token as a GitHub secret (with name NPM_TOKEN) in the repository options
```

Comment:
I don't think we should automatically un-comment this or how are you setting up the NPM_TOKEN secret for the user?

Reply:
The user will be asked for the tokens (currently npm, later also Sentry and perhaps others). Since the creator on the portal has full rights for the repo, it will be able to set the secret.

Reply:
okay :)